omni-relayer/src/utils/eth.rs

```diff
@@ -0,0 +1,41 @@
+use std::sync::Arc;
+
+use alloy::{rpc::types::Log, sol};
+use nep141_connector::Nep141Connector;
+
+sol!(
+    #[derive(Debug)]
+    event Withdraw(
+        string token,
+        address indexed sender,
+        uint256 amount,
+        string recipient,
+        address indexed tokenEthAddress
+    );
+);
+
+pub async fn process_log(connector: Arc<Nep141Connector>, log: Log) {
+    if let Ok(decoded_log) = log.log_decode::<Withdraw>() {
+        log::info!("Decoded log: {:?}", decoded_log);
+
+        let Some(tx_hash) = decoded_log.transaction_hash else {
+            log::warn!("No transaction hash in log: {:?}", log);
+            return;
+        };
+        let Some(log_index) = decoded_log.log_index else {
+            log::warn!("No log index in log: {:?}", log);
+            return;
+        };
+
+        match connector
+            .finalize_withdraw(
+                primitive_types::H256::from_slice(tx_hash.as_slice()),
```

Comment:
Maybe it's better to send tx_hash as a string to `bridge-sdk` and parse it to `H256` there. I don't think that introducing a whole crate for one type is a good idea

Reply:
> send tx_hash as a string to `bridge-sdk` and parse it to `H256` there

@kiseln What do you think about this? Can we do it like that or it would introduce a lot of mess on sdk's side?

Reply:
I'm not exactly sure. I assume it's a light weight crate and I'd rather keep strong typing. Is it possible to re-export these types from the sdk so the client doesn't have to specify the dependency?

Reply:
> Is it possible to re-export these types from the sdk so the client doesn't have to specify the dependency?

Not sure, but I'll check this. Then for this PR I'll just leave this crate